app/components/nav.tsx

```diff
@@ -14,7 +14,7 @@ export function Navbar() {
         <div className="flex items-center">
           <Link href="/" className="tracking-tight">
             <p className="text-center text-3xl font-semibold">{"KATEŘINA URBANOVÁ"}</p>
-            <p className="text-center text-base">{"Kouč pro tělo i mysl"}</p>
+            <p className="text-center text-base">{"Integrativní Kouč"}</p>
```

Comment:
FYI muzes se tady vykaslat na {} a uvozovky.
Text ti tady sezere normalne
```suggestion
            <p className="text-center text-base">Integrativní Kouč</p>
```